### ISSUE TITLE
Keyboard layout/key combination issues

### ISSUE THREAD

**Issue by nukemdu** [lem-project/lem#1569]
I use the german bone layout, where Caps-Lock is used as a sort of Right Alt key to shift to a different layer.

So when I (physically) press Ctrl-Caps-s it translates to Ctrl-/ via the selected layout. That works in Emacs for example, and basically everywhere else I've tried. Lem however reads it as a literal Ctrl-Caps-t. So it correctly gets the t key (t on the physical keyboard), but somehow also gets the Capslock key.

In short:
- Pressed (on qwerty): Ctrl-Caps-s
- Maps to (everywhere else): Ctrl-/
- Lem with SDL2 reads: Ctrl-Caps-t

This occurs in SDL2, not in ncurses, so I suspect the SDL input system is trying to read keycodes directly?

I run KDE 6.2 (wayland) on Fedora Atomic, with SBCL 2.3.11-2.fc40.
